urllib3/util/__init__.py

```diff
@@ -6,6 +6,7 @@
 from .ssl_ import (
     SSLContext,
     HAS_SNI,
+    IS_PYOPENSSL,
```

Comment:
IS_ is fine, I was just wondering if we wanted to normalize towards consistency with `HAS_SNI`